docs/source/quickstart.rst

```diff
@@ -183,32 +194,34 @@ out:
         return {DEFAULT_KEY: df}
 
 The interesting part that we need is the block at the bottom that's commented
-out. Uncomment this block and then run ``kidsfirst test`` to perform
-extraction and tranformation.
+out. Uncomment this block and then use ``kidsfirst test my_study/ingest_1
+--no_validate`` again to perform extraction and transformation without data
+validation.
 
 Run the ingest pipeline
 =======================
 
-The Warehouse Server
---------------------
+Normally you won't run without data validation, because the automatic
+validation rules will help you determine when your package is ready to ingest
+into your target service. So during real testing, it's usually best to not use
+the ``--no_validate`` option.
 
-Data that we ingest is meant to be shared, not stored locally on your machine!
-It's easy to configure the ingest system to connect to a remote warehousing
-database which will centrally store the relevant data.
+If your ingest package passes validation with the test command and you're happy
+with the result, you're ready to ingest the data into your target service using
+the ingest command::
 
-We're ready to test the ingest package again, and, since we're just seeing how
-this package works on dummy data, we don't want
-to actually upload anything to a warehouse. This won't happen if we use the
-``--no_warehouse`` option command, like so::
+    $ kidsfirst ingest my_study
 
-    $ kidsfirst test --no_warehouse my_study
+The library defaults to ingesting data into a Kids First Data Service running
+at the base url: ``http://localhost:5000``. You can change the base URL for
+this service using the ``--target_url`` command option.
 
-If the ingest package passes validation with the test command and you're happy
-with the output, you're ready to ingest the data into your target service using
-the ingest command::
+The Warehouse Server
+--------------------
 
-    $ kidsfirst ingest my_study
+Data from ingesting is meant to be shared, not stored locally on your machine!
+It's easy to configure the ingest system to connect to a remote warehousing
+database which will centrally store intermediate stage data.
```

Comment:
Why don't we mention how to export a warehouse URL so that it can be grabbed by the ingest library?

Reply:
I don't have any insight into this. Getting our warehouse db URL is very convoluted.

Reply:
Oh, I simply meant `export KF_WAREHOUSE_DB_URL=<dialect>://<user>:<password>@<host>:<port>/<database>`.